modules/arm_plugin/Dockerfile.RPi32

```diff
@@ -0,0 +1,137 @@
+FROM debian:9
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PAKAGE_UPDATES_OF 20210120_11
+
+RUN apt-get update && apt-get -y upgrade
+
+#Prerequisit installation
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        lsb-release \
+        nano \
+        wget \
+        curl \
+        nano \
+        tar \
+        bzip2 \
+        unzip \
+        cpio \
+        cifs-utils \
+        locales \
+        rsync \
+        apt-transport-https \
+        debian-archive-keyring \
+        sudo \
+        openssh-client \
+        default-jre \
+        p7zip-full \
+        software-properties-common \
+        dirmngr \
+        gnupg \
+        netcat-openbsd \
+        dh-autoreconf \
+        libcurl4-gnutls-dev \
+        libexpat1-dev \
+        gettext \
+        libz-dev \
+        libssl-dev \
+        build-essential \
+    && \
+    locale-gen en_US.UTF-8 && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN dpkg --add-architecture armhf && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends -f -o Dpkg::Options::="--force-confnew" \
+        build-essential \
+        libusb-1.0-0-dev:armhf \
+        software-properties-common \
+        crossbuild-essential-armhf \
+        zlib1g-dev \
+        libffi-dev \
+        libssl-dev \
+        chrpath \
+        libssl-dev \
+        libprotobuf-dev \
+        libprotoc-dev \
+        protobuf-compiler \
+        # For ARM CPU plugin
+        scons
+
+RUN wget https://www.cmake.org/files/v3.13/cmake-3.13.3.tar.gz && \
+    tar xf cmake-3.13.3.tar.gz && \
+    (cd cmake-3.13.3 && ./bootstrap --parallel=$(nproc --all) && make --jobs=$(nproc --all) && make install) && \
+    rm -rf cmake-3.13.3 cmake-3.13.3.tar.gz
+
+RUN curl -sSf "https://packagecloud.io/install/repositories/github/git-lfs/config_file.list?os=debian&dist=stretch&source=script" > "/etc/apt/sources.list.d/github_git-lfs.list"
+RUN curl -L "https://packagecloud.io/github/git-lfs/gpgkey" 2> /dev/null | apt-key add - &>/dev/null && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends  --allow-unauthenticated \
+        git-lfs \
+        && \
+    rm -rf /var/lib/apt/lists/*
+
+# build and install git 2.22
+RUN wget https://github.com/git/git/archive/v2.22.2.tar.gz && \
+    tar xf v2.22.2.tar.gz && \
+    cd git-2.22.2 && \
+    make configure && \
+    ./configure --prefix=/usr && \
+    make all && \
+    make install && \
+    rm -rf /git-2.22.2 && \
+    rm /v2.22.2.tar.gz
```

Comment:
Comments: what? => why?

Please add comment describing why system "git" package version is not enough (which features are required?)

Reply:
If system git package is used than rate limit exceeded when openvino repo is being cloned:
```
Error downloading object: docs/img/benefit_ncs2.png (ba09c1c): Smudge error: Error downloading docs/img/benefit_ncs2.png (ba09c1c8b748936350df0999b1a926a41ca040f171754bd7884950fe13402e4c): batch response: Rate limit exceeded: https://github.com/openvinotoolkit/openvino.git/info/lfs/objects/batch
```
I don't know what exactly feature\fix of git 2.22.2 solves this issue. It's better to ask @terfendail but he's on sick leave.

Reply:
It was found that system `git lfs` solves this issue. Removed git 2.22 from Dockerfile and replace `git` with `git lfs` to clone OpenVINO repository.